frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -237,7 +237,7 @@ export const CreateSubscriptionYAML: React.SFC<CreateSubscriptionYAMLProps> = (p
       ]}
     >
       {/* FIXME(alecmerdler): Hack because `Firehose` injects props without TypeScript knowing about it */}
-      <Create {...props as any} />
+      <Create {...(props as any)} />
```

Comment:
Another surprise pass on the spread `...props` rule.

Reply:
I don't know that we currently have a way to fix the use of spread operators here without updating Firehose.